src/test/java/uk/gov/hmcts/reform/civil/notification/EvidenceUploadApplicantNotificationHandlerTest.java

```diff
@@ -53,11 +45,7 @@ void setup() {
         void shouldNotifyApplicantSolicitor_whenInvoked() {
             //given: case where applicant solicitor has email as applicantsolicitor@example.com
             CaseData caseData = CaseDataBuilder.builder().atStateClaimDetailsNotified().build();
-            CallbackParams params = CallbackParamsBuilder.builder().of(ABOUT_TO_SUBMIT, caseData).build();
-
-            //when: about to submit is called for callback handler
-            handler.handle(params);
-
+            handler.notifyApplicantEvidenceUpload(caseData);
```

Comment:
//when: Evidence for Applicant is uploaded

Reply:
done